projects/storefrontapp/src/environments/environment.prod.ts

```diff
@@ -9,5 +9,5 @@ export const environment: Environment = {
   cds: buildProcess.env.CX_CDS,
   b2b: buildProcess.env.CX_B2B,
   cdc: buildProcess.env.CX_CDC,
-  tms: buildProcess.env.CX_TMS,
+  tracking: buildProcess.env.CX_TRACKING,
```

Comment:
As we are adding those env build flags everywhere, shouldn't we also create e2es for this feature?
The other way around: is this flag justified or we rather should enable it by default?

Reply:
Enabled the feature by defualt